tensorflow/compiler/jit/flags.cc

```diff
@@ -78,6 +89,21 @@ void AppendMarkForCompilationPassFlagsInternal(std::vector<Flag>* flag_list) {
       Flag("tf_xla_max_cluster_size",
            &mark_for_compilation_flags->tf_xla_max_cluster_size,
            "Maximum number of operators in an XLA compilation."),
+      Flag("tf_xla_supported_ops",
+           &mark_for_compilation_flags->tf_xla_supported_ops,
+           "Limit the operations clustered by XLA to these operations. "
+           "If multiple, separate them with commas. Shortcuts: "
+           " PW: All point-wise operations."
+           " RED: All reduction operations."
+           " SMALL: Mixed small operations."
```

Comment:
Small is like Shape, Rank, Range and a few others I didn't know in which category to put. I could split that section into 'SMALL' and MIXED. Do you have a better idea of how to split that?

Reply:
Since you said you rarely used the subcategories of FUSIBLE anyway, I wouldn't split them further. I was just wondering in what way the TF ops are SMALL. If this is a category for ops that just wouldn't fit anywhere else, I'd call it MISC to make this obvious.

Reply:
Done